ntia_conformance_checker/base_checker.py

```diff
@@ -724,6 +728,26 @@ def get_components_without_versions(
 
         return []
 
+    def _get_all_components_without_infos(self) -> List[Tuple[str, List[str]]]:
```

Comment:
nit and it's subjective: the `infos` part is throwing me off. I'd expect just `info`. Minor. Just thinking out loud. No need to change it if it doesn't trip you up.